samples/BenchmarkDotNet.Samples/IntroNuGet.cs

```diff
@@ -26,22 +27,26 @@ public Config()
                 var baseJob = Job.MediumRun;
 
                 string[] targetVersions = [
-                    "13.0.3",
-                    "13.0.2",
-                    "13.0.1"
+                    "9.0.3",
+                    "9.0.4",
+                    "9.0.5",
                 ];
 
                 foreach (var version in targetVersions)
                 {
-                    AddJob(baseJob.WithNuGet("Newtonsoft.Json", version)
-                                  .WithId(version));
+                    AddJob(baseJob.WithNuGet("System.Collections.Immutable", version)
+                                  .WithId("v"+version));
                 }
             }
         }
 
+        private static readonly Random rand = new Random(Seed: 0);
+        private static readonly double[] values = Enumerable.Range(1, 10_000).Select(x => rand.NextDouble()).ToArray();
+
         [Benchmark]
-        public void SerializeAnonymousObject()
-            => JsonConvert.SerializeObject(
-                new { hello = "world", price = 1.99, now = DateTime.UtcNow });
+        public void ToImmutableArrayBenchmark()
+        {
+            var results = values.ToImmutableArray();
```

Comment:
I think we can improve the test by asserting the assembly version of the package. See multiple frameworks tests for how to pass expected values to compare in the benchmark code.

Reply:
Oops wrong file.